azurelinuxagent/common/version.py

```diff
@@ -196,10 +196,9 @@ def has_logrotate():
 
 AGENT_NAME = "WALinuxAgent"
 AGENT_LONG_NAME = "Azure Linux Agent"
-# Setting the version to 9.9.9.9 for testing purposes.
+# Setting the version to 9.9.9.9 to ensure DCR always uses this version and never auto-updates.
 # Replace this with the actual agent version on release.
-# Current Agent Version = 2.2.54.2
-AGENT_VERSION = '9.9.9.9'
+AGENT_VERSION = '2.3.0.2'  # (current agent version = 2.3.0.2)
```

Comment:
I will do a new PR to set the version to 9.9.9.9 on develop and update the comment